js_modules/dagit/packages/core/src/nav/RepoNavItem.tsx

```diff
@@ -168,14 +168,22 @@ const SingleRepoNameLink = styled(Link)`
   }
 `;
 
+const ReloadTooltip = styled(Tooltip)`
+  && {
```

Comment:
not sure I've seen this one, what does the double `&&` do? Can this just be a single &? and in that case could we boil this down to just the display:block line?

Reply:
Each `&` increases the specificity for the rule. In this case I'm up against a `span.className` with the `display: inline-block` I want to defeat, so this gives me a `.className.className`.